Helper/Order.php

```diff
@@ -417,13 +417,14 @@ private function setPaymentMethod($quote)
      * @param \stdClass $transaction
      * @param OrderModel $order
      * @param Quote $quote
+     * @throws \Exception
      */
     private function adjustTaxMismatch($transaction, $order, $quote)
     {
-        $boltTaxAmount = round($transaction->order->cart->tax_amount->amount / 100, 2);
-        $boltTotalAmount = round($transaction->order->cart->total_amount->amount / 100, 2);
-
-        $orderTaxAmount = round($order->getTaxAmount(), 2);
+        $precision = CurrencyUtils::getPrecisionForCurrencyCode("USD");
+        $boltTaxAmount = round(CurrencyUtils::toMajor($transaction->order->cart->tax_amount->amount, "USD"), $precision);
```

Comment:
can we make toMajor round to the precision?

Reply:
done